modules/registry-exposing-secure-registry-manually.adoc

```diff
@@ -5,126 +5,64 @@
-+
-----
-$ oc get route/pod-registry -o yaml
-apiVersion: v1
-kind: Route
-metadata:
-  name: podman-registry
-spec:
-  host: <host> <1>
-  to:
-    kind: Service
-    name: podman-registry <2>
-  tls:
-    termination: passthrough <3>
-----
-<1> The host for your route. You must be able to resolve this name externally by
-DNS to the router's IP address.
-<2> The service name for your registry.
-<3> Specifies this route as a passthrough route.
-+
-[NOTE]
-====
-Re-encrypt routes are also supported for exposing the secure registry.
-====
+You can expose the route by using `DefaultRoute` or custom routes.
 
-.. If it does not exist, create the route with the `oc create route passthrough`
-command, specifying the registry as the route’s service. By default, the name of
-the created route is the same as the service name:
+Expose the registry using `DefaultRoute`:
 
-... Get the *podman-registry* service details:
+. Set `DefaultRoute` to `True`:
 +
 ----
-$ oc get svc
-NAME              CLUSTER_IP       EXTERNAL_IP   PORT(S)                 SELECTOR                  AGE
-podman-registry   172.30.69.167    <none>        5000/TCP                podman-registry=default   4h
-kubernetes        172.30.0.1       <none>        443/TCP,53/UDP,53/TCP   <none>                    4h
-router            172.30.172.132   <none>        80/TCP                  router=router             4h
+$ oc patch config.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge
 ----
-
-... Create the route:
 +
-----
-$ oc create route passthrough    \
-    --service=podman-registry    \//<1>
-    --hostname=<host>
-route "podman-registry" created     <2>
-----
-<1> Specifies the registry as the route's service.
-<2> The route name is identical to the service name.
-
-. Trust the certificates being used for the registry on your host
-system to allow the host to push and pull images. The certificates referenced
-were created when you secured your registry:
+. Log in with Podman:
 +
 ----
-$ sudo mkdir -p /etc/podman/certs.d/<host>
-$ sudo cp <ca_certificate_file> /etc/podman/certs.d/<host>
-$ sudo systemctl restart podman
+HOST=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')
+podman login -u $(oc whoami) -p $(oc whoami -t) --tls-verify=false $HOST <1>
 ----
+<1> `--tls-verify=false` is needed if the cluster's default certificate for routes
+is untrusted. You can set a custom, trusted certificate as the default
+certificate with the xref:../networking/ingress-operator.adoc#configuring-ingress[Ingress Operator].
 
-. Log in to the registry using the information from securing the registry.
-However, this time point to the host name used in the route rather than your
-service IP. When logging in to a secured and exposed registry, make sure you
-specify the registry in the `podman login` command:
+
+Expose the registry using custom routes:
+
+. Create a secret:
```

Comment:
```suggestion
. Optional - create a secret with your route's TLS keys:
```

Reply:
Thanks! I added "Create a secret with your route's TLS keys" and have in a line following the command a statement that it's optional. Is that okay?